docs/components/board/_index.md

```diff
@@ -428,55 +428,6 @@ err := myBoard.Status(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
-### ModelAttributes
-
-Get the attributes related to the model of this board.
-
-{{< tabs >}}
-{{% tab name="Python" %}}
-
-**Parameters:**
-
-- None
-
-**Returns:**
-
-- [(Attributes)](https://python.viam.dev/autoapi/viam/components/board/index.html#viam.components.board.Board.Attributes): Attributes related to the model of this board.
```

Comment:
:eyes: That link shouldn't exist any more but it does! and it does because I forgot to delete the `Attributes` class even though it's no longer used. I'll go make a follow-up PR, and when that's in, the docs at this link should automatically update to match. I'm glad I saw this and noticed my mistake.

Reply:
Linking things up: the follow-up PR is https://github.com/viamrobotics/viam-python-sdk/pull/501